CHANGELOG.md

```diff
@@ -1,3 +1,40 @@
+## [13.12.0](https://github.com/commercetools/merchant-center-application-kit/compare/v13.11.0...v13.12.0) (2019-06-27)
+
+#### 🐛 Type: Bug
+
+- `jest-stylelint-runner`
+  - [#789](https://github.com/commercetools/merchant-center-application-kit/pull/789) fix(jest-stylelint-runner): include run file ([@montezume](https://github.com/montezume))
```

Comment:
maybe also link to the original `jest-stylelint-runner` . commit? describing how it's new feature

Reply:
Moved things around. 
35ab210

Reply:
Taking a leap faith now :)